### PR TITLE
ADEN-7701 Register Roadblock from ad-engine

### PR DESCRIPTION
**Pull request by fraszczakszymon**
## Links

* http://wikia-inc.atlassian.net/browse/ADEN-7701

### CODE REVIEW

File: app/modules/ads/templates/roadblock-config.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+export const getConfig = () => (
+  {
+    slotsToEnable: [
+      'incontent_boxad_1',
+      'mobile_in_content',
+    ],
+  }
+);
+
+export default {
```
Comment (jacekbj) [inline app/modules/ads/templates/roadblock-config.js:8]: Out of curiosity, can it be
```
export default function getConfig() {
  return {
    slotsToEnable: [
      'incontent_boxad_1',
      'mobile_in_content',
    ],
  }
}
```?

File: app/modules/ads/templates/roadblock-config.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+export const getConfig = () => (
+  {
+    slotsToEnable: [
+      'incontent_boxad_1',
+      'mobile_in_content',
```
Comment (jacekbj) [inline app/modules/ads/templates/roadblock-config.js]: What about bottom leaderboard? How is it enabled?
I know I sometimes ask stupid/repeated questions and thank you for your patience!

File: app/modules/ads/templates/roadblock-config.js (hunk 1 above)
Comment (fraszczakszymon) [reply app/modules/ads/templates/roadblock-config.js:8]: You're right 👍

File: app/modules/ads/templates/roadblock-config.js (hunk 2 above)
Comment (fraszczakszymon) [reply app/modules/ads/templates/roadblock-config.js]: There should be only +1 slot in roadblock (along with TLB). Nef decided to enable incontent one (when AE3 is enabled)

**Review by jacekbj — APPROVED** [review]

File: app/modules/ads/templates/roadblock-config.js (hunk 1 above)
Comment (fraszczakszymon) [reply app/modules/ads/templates/roadblock-config.js:8]: Nope, I changed it a little bit.

**Pull request merged**